content/en/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md

```diff
@@ -3,7 +3,7 @@ reviewers:
 - bsalamat
 title: Scheduler Performance Tuning
 content_type: concept
-weight: 70
+weight: 80
```

Comment:
Instead of changing the weight, you might have assign a different weight value to new pages. The purpose of assigning weights as 30, 40, 50 today is to make it easier to insert new pages that can have weights like 35, 37 etc.

Reply:
Thanks i will ensure to use this for future enhancements..